pkg/diagnostics/v2/client_test.go

```diff
@@ -198,7 +198,7 @@ func TestCreateHappyPath(t *testing.T) {
 	defer ts.Close()
 
 	c := NewClient(pluginutil.HTTPClient(ts.URL))
-	id, err := c.Create()
+	id, err := c.Create(Options{})
```

Comment:
We definitely need tests for the correct parsing of the flags.

Reply:
Tests added